src/ringmesh/basic/common.cpp

```diff
@@ -112,8 +112,8 @@ namespace RINGMesh
         Logger::out(
             "", "RINGMesh-dev <georessources-ringmesh-dev@univ-lorraine.fr> " );
         Logger::out( "", "You can have access to the full open-source ",
-            "code through its Bitbucket repository: " );
-        Logger::out( "", "https://bitbucket.org/ring_team/ringmesh" );
+            "code through its GitHub repository: " );
+        Logger::out( "", "https://github.com/ringmesh/RINGMesh" );
```

Comment:
I think it is also working with lower-case. This might be better for links...

Reply:
I do not know. If there is no problem with upper-case for links, I think it is better to keep RINGMesh typography.

Reply:
OK